lib/core/src/lib/datatable/components/datatable/datatable.component.spec.ts

```diff
@@ -1694,6 +1694,11 @@ describe('Column Resizing', () => {
 
     const getResizeHandler = (): HTMLDivElement => fixture.debugElement.nativeElement.querySelector('.adf-datatable__resize-handle');
 
+    const getResizeHandlersCount = (): number => {
+        const resizeHandlers = fixture.debugElement.nativeElement.querySelectorAll('.adf-datatable__resize-handle');
+        return resizeHandlers.length;
+    }
```

Comment:
Double empty lines and missing semicolon

Reply:
Fixed